sct-commons/src/main/java/org/lfenergy/compas/sct/commons/scl/ied/DOIAdapter.java

```diff
@@ -153,6 +159,21 @@ private String createInRefValTestString(TExtRef extRef) {
         return new ObjectReference(extRef, ExtrefTarget.SRC_CB).getReference();
     }
 
+    private ResumedDataTemplate createDaiFilter(String daName) {
```

Comment:
This instance of ResumedDataTemplate can be created in ResumedDataTemplate class
```java
    public ResumedDataTemplate (AbstractLNAdapter lnAdapter, String doName, String daName) {
        this.lnClass = lnAdapter.getLNClass();
        this.lnInst = lnAdapter.getLNInst();
        this.prefix = lnAdapter.getPrefix();
        this.lnType = lnAdapter.getLnType();
        this.doName = new DoTypeName(doName);
        this.daName = new DaTypeName(daName);
    }
```